lib/graph_records.rb

```diff
@@ -0,0 +1,450 @@
+# frozen_string_literal: true
```

Comment:
I feel like this should also be in `app/lib` ... at this point we're going to treat these ops tools as part of the app in terms of ops support.